src/test/java/com/easypost/Fixture.java

```diff
@@ -410,4 +410,25 @@ public static Map<String, Object> taxIdentifier() {
     public static String webhookUrl() {
         return "http://example.com";
     }
+
+    /**
+     * RMA form options.
+     *
+     * @return RMA form options.
+     */
+    public static Map<String, Object> rmaFormOptions() {
+        final int units = 8;
```

Comment:
# Reviewer Note:

We have this variable to avoid `is a magic number` from CI style enforcer.